src/pages/main_layout.cr

```diff
@@ -20,36 +20,192 @@ abstract class MainLayout
+        complex JSON usually takes only a few milliseconds.
+        TEXT
+
+        content_block "Batteries included", <<-TEXT
+        Authentication, asset management, CORS, database ORM, and more can
+        all be included when creating a new Lucky project.
+        TEXT
+
+        content_block "Fewer bugs. More joy", <<-TEXT
+        Instead of finding bugs in QA or in production, Lucky is designed to
+        catch as many bugs as possible at compile time.
+        TEXT
+      end
+    end
+  end
+
+  private def content_block(title, body)
+    div class: "w-full md:w-1/3 sm:px-0 md:px-8 my-5 md:my-0" do
+      h3 title, class: "mb-3 text-black font-bold text-base"
+      para body, class: "leading-loose text-sm text-grey-darker"
+    end
+  end
+
+  private def render_freeform_text
+    render_markdown_content <<-TEXT
+    ## Why is modern web development so complicated?
+
+    **There are more choices in tech than ever**, and with it comes more questions and more complexity.
+
+    Have you ever thought:
+
+    * "Why do I have to write **so much boilerplate**?"
+    * "How do **so many bugs make it to production**?"
+    * "Why is my app **so slow**!?"
+    * "Why do I have to choose between fast prototyping and long term maintainability?"
+
+    ## What if you could create something quickly **and** maintain it for years to come?
+
+    Imagine this:
+
+    * Generators for quick code creation
+    * Built in authentication (Optional)
+    * Secure by default
+    * Built in Webpack config to get a SPA off the ground fast (Optional)
+    * Insanely fast execution so users aren’t left waiting
+    * Powerful type system that helps catch bugs before they ever hit production
+
+    If this sounds interesting, [let's get started](#{Guides::GettingStarted::Installing.path}).
+
+    If you're not quite ready to dive in but want to stay up to date,
+    [subscribe to our blog](#{Blog::Index.path}), or [follow us on
+    Twitter](https://twitter.com/luckyframework).
+
+    ## What does Lucky look like?
+
+    ### JSON API
+
+    ```crystal
+    class Api::Users::Show < ApiAction
+      get "/api/users/:user_id" do
+        json user_json
+      end
+
+      private def user_json
+        user = UserQuery.find(user_id)
+        {name: user.name, email: user.email}
+      end
+    end
+    ```
+
+    ### Database
+
+    ```crystal
+    # Set up the model
+    class User < BaseModel
+      table :users do
+        column last_active_at : Time
+        column last_name : String
+      end
+    end
+
+    # Add some methods to help query the database
+    class UserQuery < User::BaseQuery
+      def recently_active
+        last_active_at.gt(1.week.ago)
+      end
+
+      def sorted_by_last_name
+        last_name.lower.desc_order
+      end
+    end
+
+    # Query the database
+    UserQuery.new.recently_active.sorted_by_last_name
+    ```
+
+    ### Rendering HTML
+
+    ```crystal
+    class Users::Index < BrowserAction
+      route do
```

Comment:
Maybe this should just be `get "/users"` just in case we do end up making that other change?

Reply:
Great idea 👍

Reply:
This is in reference to https://github.com/luckyframework/lucky/issues/789